assets/pages/new-contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -92,6 +102,7 @@ function FormFields(props: PropTypes) {
         placeholder="example@domain.com"
         icon={<SvgEnvelope />}
         onInput={props.updateEmail}
+        onChange={props.checkIfEmailHasPassword}
```

Comment:
I would put this in `onInput', and only call the API if the email is a valid email address, for the reasons listed in my general comment on this PR.